RELEASES.md

```diff
@@ -19,47 +38,50 @@
       If changes should not be backported, keep them in the _changelogs/head.asciidoc_ file.
     * Patch version: Add new section to existing release notes. ([Sample PR](https://github.com/elastic/apm-server/pull/2064/files))
 
-    Create PR in `main` and backport.
+    Create the changelog PR in `main` and backport to the release branch.
```

Comment:
IIUC, this will cause the `head.asciilog` file to be included in the release branch. Is that intentional?